pages/agent/v3/aws/secrets_manager.md

```diff
@@ -0,0 +1,80 @@
+# Storing your Buildkite Agent token in AWS Secrets Manager
+
+The Elastic CI Stack for AWS supports reading a Buildkite Agent token from
+the AWS Systems Manager Parameter Store. The token can be stored in a plaintext
+parameter, or encrypted with a KMS Key for access control purposes. You can also store your Buildkite Agent token using AWS Secrets Manager if
+you need the advanced functionality it offers over the Parameter
+Store.
+
+For example, AWS Secrets Manager can automatically rotate and
+revoke secrets using Lambda functions, and replicate secrets across multiple
+regions in your account.
+
+To store your Buildkite Agent token as an AWS Secrets
```

Comment:
@plaindocs I’ve tweaked this paragraph, could I ask you to have another look before I merge it 🙏🏼